### ISSUE TITLE
Ignored Capabilities

### ISSUE THREAD

**Issue by codeclinic** [sbrajesh/buddyblog#10]
Hi,

It appears that the following options are being ignored:

`Which capability is required for pusblishing?`
`Which capability is required for creating post?`

Regardless of the capability entered it seems all users can create and publish posts.

**Comment by sbrajesh** [comment]
Hi,
Thank you for posting.
Can you please provide more details?
- Are you using custom post type?
  -Which capability did you set and check with?
  -Which version of WordPress, BuddyPress and BuddyBlog are you using?

Thank you
Brajesh

**Comment by codeclinic** [comment]
I'm using the latest version of all WP, BP & BB.

I'm using a custom post type and a specifically a custom capability as the required capability level. However all users can still post.

**Comment by sbrajesh** [comment]
Hi,
It seems it is most probably an issue with the custom capability usage. Can you please select the "post" post type and give some rights/capability and check if it works? That will give us an Idea where the issue may be.

**Comment by codeclinic** [comment]
I'm not entirely sure what you are asking me to do. However, as a test I gave my user the default WordPress user role of 'Subscriber'. Then in Buddyblog I set the following settings:

Which capability is required for pusblishing?                                             publish_posts
Which capability is required for creating post?                                           publish_posts
Which capability can edit post in backend(WordPress Dashboard)?          publish_posts

However, it would appear that anyone can still post items.
